src/include/execution/vm/bytecode_handlers.h

```diff
@@ -67,6 +67,14 @@ ALL_TYPES(COMPARISONS);
 
 VM_OP_HOT void OpNot(bool *const result, const bool input) { *result = !input; }
 
+VM_OP_HOT void OpNotSql(terrier::execution::sql::BoolVal *result, terrier::execution::sql::BoolVal *input) {
```

Comment:
Move this logic into a new function in `ComparisonFunctions::NotBoolVal()`. Make sure that function is implemented in the header, too.

Try to avoid putting logic in these handlers, unless it's a "simple" bytecode (like `OpNot`). This allows them to be tested separately, and groups common SQL logic together.